crates/divan_compat/src/compat/mod.rs

```diff
@@ -0,0 +1,35 @@
+// Used by generated code. Not public API and thus not subject to SemVer.
+#[doc(hidden)]
+#[path = "private.rs"]
+pub mod __private;
```

Comment:
This is not super clear what will use this module (from what I understood it will be the expanded macro from divan).

Reply:
Also since this mod is fairly small and to avoid back and forth in the module tree it would be a bit simpler to declare the mod inline here.